src/components/FilterBarPopup/FilterBarPopup.stories.tsx

```diff
@@ -0,0 +1,399 @@
+    return (
+      multiSelectItems &&
+      multiSelectItems.map((multiSelect) => (
+        <MultiSelect
+          buttonText={multiSelect.name}
+          key={multiSelect.id}
+          id={multiSelect.id}
+          items={multiSelect.items}
+          selectedItems={selectedItems}
+          isBlockElement={isBlockElement}
+          onChange={(e) => onChange(e.target.id, multiSelect.id)}
+          onMixedStateChange={(e) => {
+            return onMixedStateChange(
+              e.target.id,
+              multiSelect.id,
+              multiSelect.items
+            );
+          }}
+          onClear={() => onClear(multiSelect.id)}
+          width={multiSelectWidth}
+        />
+      ))
+    );
+  };
+
+  const renderFilterComponents = ({ layout, width }) => {
+    return (
+      <MultiSelectGroup
+        id="multiselect-group"
+        labelText="MultiSelect Group"
+        layout={layout}
+        multiSelectWidth={width}
+        renderMultiSelect={renderMultiSelect}
+      />
+    );
+  };
+
+  return (
+    <FilterBarPopup
+      id={args.id}
+      heading={args.heading}
+      onClear={onClearAll}
+      onSubmit={() => console.log(selectedItems)}
+      selectedItems={selectedFilterItems}
+      renderChildren={renderFilterComponents}
+    />
+  );
+};
+
+const FilterBarPopupHeadingStory = () => {
+  const { onChange, onMixedStateChange, selectedItems, onClear } =
+    useMultiSelect();
+
+  const renderMultiSelect = ({ isBlockElement, multiSelectWidth }) => {
+    return (
+      multiSelectItems &&
+      multiSelectItems.map((multiSelect) => (
+        <MultiSelect
+          buttonText={multiSelect.name}
+          key={multiSelect.id}
+          id={multiSelect.id}
+          items={multiSelect.items}
+          selectedItems={selectedItems}
+          isBlockElement={isBlockElement}
+          onChange={(e) => onChange(e.target.id, multiSelect.id)}
+          onMixedStateChange={(e) => {
+            return onMixedStateChange(
+              e.target.id,
+              multiSelect.id,
+              multiSelect.items
+            );
+          }}
+          onClear={() => onClear(multiSelect.id)}
+          width={multiSelectWidth}
+        />
+      ))
+    );
+  };
+
+  const renderFilterComponents = ({ layout, width }) => {
+    return (
+      <MultiSelectGroup
+        id="multiselect-group"
+        labelText="MultiSelect Group"
+        layout={layout}
+        multiSelectWidth={width}
+        renderMultiSelect={renderMultiSelect}
+      />
+    );
+  };
+
+  return (
+    <VStack align="left">
+      <Box>Custom H3 Heading</Box>
+      <FilterBarPopup
+        heading={<Heading level="h3">Custom H3 Heading</Heading>}
+        renderChildren={renderFilterComponents}
+      />
+      <Box>Custom H5 Heading</Box>
+      <FilterBarPopup
+        heading={<Heading level="h5">Custom H5 Heading</Heading>}
+        renderChildren={renderFilterComponents}
+      />
+      <Box>No Heading</Box>
```

Comment:
Now that we will have a default value for the `heading` props, change this to `Default heading`.